lib/shared/generate-circuit-json.ts

```diff
@@ -37,7 +38,11 @@ export async function generateCircuitJson({
 }: GenerateCircuitJsonOptions) {
   debug(`Generating circuit JSON for ${filePath}`)
 
-  const runner = new CircuitRunner()
+  const worker = await createCircuitWebWorker({
```

Comment:
This is a major breaking change, nodejs does not have webworker support and webworkers should not be used outside the browser

Reply:
Let me look more into this